package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "dev": "node server.js",
-    "dev:debug": "node --inspect server.js",
+    "dev:debug": "node --inspect-brk server.js",
```

Comment:
Should probably just call this `debug` and maybe mention it in the README or docs contrib guide...

Reply:
Name updated. And it's already mentioned in https://dvc.org/doc/user-guide/contributing/docs#debugging.